include/rootston/seat.h

```diff
@@ -60,6 +69,8 @@ void roots_seat_configure_xcursor(struct roots_seat *seat);
 
 bool roots_seat_has_meta_pressed(struct roots_seat *seat);
 
+struct roots_view *roots_seat_get_focused_view(struct roots_seat *seat);
```

Comment:
Since views are primarily what is focused, I think we can change the name to `roots_seat_get_focused()`.